queries.sql

```diff
@@ -26,3 +28,76 @@ SELECT * FROM animals WHERE name NOT LIKE 'Gabumon';
 -- Find all animals with a weight between 10.4kg and 17.3
 SELECT * FROM animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
 
+-- Milestone: Vet clinic database: query and update animals table
+
+-- How many animals are there?
+SELECT COUNT(id) FROM animals;
+-- Ans: 10
+
+-- How many animals have never tried to escape?
+SELECT COUNT(id) FROM animals WHERE escape_attempts < 1;
+-- Ans: 2
+
+-- What is the average weight of animals?
+SELECT AVG(weight_kg) FROM animals;
+-- Ans: 15.55
+
+-- Who escapes the most, neutered or not neutered animals?
+SELECT * FROM animals WHERE neutered = true OR neutered = false ORDER BY escape_attempts DESC LIMIT 1;
```

Comment:
Hello @olawale-o ,

We missed a small mistake in your query.  The project is still approved but please note that here you are looking for one specific animal who escaped the most.

And the task was to compare two groups neutered and not neutered animals.
You could do that by comparing the average number of attempts in both groups.

Do you want to give it a try?